src/middlewares/roleCheck.js

```diff
@@ -0,0 +1,36 @@
+import jwt from 'jsonwebtoken';
```

Comment:
@Mucyo-Aime-Christian  I don't think we need this middleware you can use the authentication middleware to check if the user exists and use isAllowed middleware to check if the user is the trip admin.